tests/requirements.txt

```diff
@@ -15,9 +15,7 @@
 packaging
 pytest
 PyYAML
-importlib-metadata<5.0
 
 # pin to version of TF used in prod
-gpflux == 0.2.3
-tensorflow ~= 2.4.1
-tensorflow-probability ~= 0.12.0
+tensorflow ~= 2.8.0
```

Comment:
Why is this `~=2.8.0` and not `~=2.5.0`?

Reply:
Because "product" is now using 2.8, so we want that to be out main test target. I think there's already an issue to increase the number of versions we test against (we could at least add a "minimal" target), but I'd like to postpone doing that until after I've fixed the memory issues we're seeing.